node/src/test/scala/com/wavesplatform/http/UtilsRouteSpec.scala

```diff
@@ -674,6 +680,107 @@ class UtilsRouteSpec extends RouteSpec("/utils") with RestAPISettingsHelper with
     }
   }
 
+  routePath("/script/evaluate/{address}") in {
```

Comment:
add tests for different types of function args: String, ByteVector, List[String|ByteVector|Boolean|Int]